File: stecuanhueygarcia6.c
// Santiago Tecuanhuey Garcia 
//
// Lab 6 - C Version 
//
//
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(void){
	
		
	double x = 19.31;
	int y = 7;

	double endResult;
	int endIntResult;

	double dubX = 19.31;
	double dubY = 7.00;
	
	int intX = 1931;
	int intY = 7;

	// The first evaluation(for both double and int).
	printf("The first evaluation (double and int) is: ");
	endResult = x + y * x/ y - x;
	printf("%lf\n",endResult);

	// The second evaluation(for both double and int).
        printf("The second evaluation (double and int) is: ");
        endResult = -x - y /x * y + x;
        printf("%lf\n",endResult);

	// The third evaluation(for both double and int).
        printf("The third evaluation (double and int) is: ");
        endResult = x + y - x / y;
        printf("%lf\n",endResult);

	printf("--------------------------------------------------\n");

	//The first evaluation(for only int within expression).
	printf("The first evaluation (for only int) is: ");
	endIntResult = x + y * x/ y - x;
	printf("%d\n",endIntResult);

	//The second evaluation(for only int within expression).
        printf("The second evaluation (for only int) is: ");
        endIntResult = -x - y /x * y + x;
        printf("%d\n",endIntResult);

	//The third evaluation(for only int within expression).
        printf("The third evaluation (for only int) is: ");
        endIntResult = x + y - x / y;
        printf("%d\n",endIntResult);

	printf("--------------------------------------------------\n");

	//The first evaluation(for only double within expression).
	printf("The first evaluation (for only double) is: ");
	endResult = x + y * x/ y - x;
        printf("%lf\n",endResult);

	//The second evaluation(for only double within expression).
	printf("The second evaluation (for only double) is: ");
	endResult = -x - y /x * y + x;
        printf("%lf\n",endResult);

	//The third evaluation(for only double within expression).
	printf("The third evaluation (for only double) is: ");
	endResult = x + y - x / y;
        printf("%lf\n",endResult);


	return EXIT_SUCCESS;
}

File: stecuanhueygarcia6.java

// Santiago Tecuanhuey Garcia 
//
// Lab 6 - Java 
//
//
//
//
public class stecuanhueygarcia6{
	public static void main (String[] args){

	System.out.println("Hello welcome.");

	double x = 19.31;
	int y = 7;

	double dubX = 19.31;
	double dubY = 7.00;

	int intX = 1931;
	int intY = 7;

	// First Expression Evaluation(for both int and double within expression.)
	System.out.println("The first (for both int and double) expression evaluation is: "); 
	System.out.println(x + y * x / y - x);

	// Second Expression Evaluation(for both int and double within expression.)
        System.out.println("The second (for both int and double) expression evaluation is: ");
        System.out.println(-x - y / x * y + x);

	// Third Expression Evaluation(for both int and double within expression.)
        System.out.println("The third (for both int and double) expression evaluation is: ");
        System.out.println(x + y - x / y);

	System.out.println("----------------------------------------");

	// First Expression Evaluation(for only int within expression.)
        System.out.println("The first (for only int) expression evaluation is: ");
        System.out.println(intX + intY * intX / intY - intX);

        // Second Expression Evaluation(for only int within expression.)
        System.out.println("The second (for only int) expression evaluation is: ");
        System.out.println(- intX - intY / intX * intY + intX);

        // Third Expression Evaluation(for only int within expression.)
        System.out.println("The third (for only int) expression evaluation is: ");
        System.out.println(intX + intY - intX / intY);

	System.out.println("----------------------------------------");

	 // First Expression Evaluation(for only double within expression.)
        System.out.println("The first (for only double) expression evaluation is: ");
        System.out.println( dubX + dubY * dubX / dubY - dubX);

        // Second Expression Evaluation(for only double within expression.)
        System.out.println("The second (for only double) expression evaluation is: ");
        System.out.println(- dubX - dubY / dubX * dubY + dubX);

        // Third Expression Evaluation(for only double within expression.)
        System.out.println("The third (for only double) expression evaluation is: ");
        System.out.println(dubX + dubY -dubX / dubY);


	}
}
